### DISCUSSION TITLE
Is it possible to remove a json object with given json-pointer?

### DISCUSSION THREAD

**Discussion by JHerrKodo** [nlohmann/json#3369] [Q&A] [answered]
_clear()_ and _erase()_ only removing the value of a json object, but i want to remove the object itself.

**Comment by gregmarr** [comment, marked as answer]
I think in that case you want to split the json-pointer into parent and child by popping the last element of the path, and remove the child from the parent.

**Reply by JHerrKodo** [reply]
How do i pop back the last element?

**Reply by falbrechtskirchinger** [reply]
Try `json_pointer::pop_back()`. https://json.nlohmann.me/api/json_pointer/pop_back/

**Reply by JHerrKodo** [reply]
Thank you. I did not recognize that there is a tab for json_pointer, in the documentation.
